### PR TITLE
bscript.compile - minimal -1 stack element

### PR DESCRIPTION
**Pull request by afk11**

### CODE REVIEW

**Comment by dcousens** [comment]
ACK

File: test/script.js [hunk 1]
```diff
@@ -45,13 +45,11 @@ describe('script', function () {
     fixtures.valid.forEach(function (f) {
       it('returns ' + !!f.stack + ' for ' + f.asm, function () {
         var script = bscript.fromASM(f.asm)
-        var chunks = bscript.decompile(script)
 
-        try {
-          var stack = bscript.toStack(chunks).map(function (x) { return x.toString('hex') })
-          assert.deepEqual(stack, f.stack)
-        } catch (e) {
```
Comment (dcousens) [inline test/script.js]: The point of the `try/catch` here was to ensure that no behaviour changed

**Comment by dcousens** [comment]
Also,  next time please just either make a PR to merge this into the *other* PR... or add to it directly.

For me to edit this, I'd have submit a PR to your branch in your repository, rather than to a branch... in this repository.

File: test/script.js (hunk 1 above)
Comment (afk11) [inline test/script.js]: I'll add it back!

**Comment by afk11** [comment]
Woops, yeah meant to target that one. if you're happy to merge both I'll leave this open, but can resubmit

**Comment by dcousens** [comment]
@afk11 this one is rebased on the other, so its fine :+1:

**Comment by afk11** [comment]
for posterity, see also: https://github.com/bitcoinjs/bitcoinjs-lib/pull/731

**Pull request merged**